docs/pages/blog/making-customizable-components.md

```diff
@@ -6,37 +6,34 @@ authors: ['alexfauquette']
 
-This article reviews several different approaches to customizing components, as well as the various tradeoffs associated with each method.
+Managing these contradictory needs only becomes more difficult as component complexity increases.
+
+This article reviews several different approaches that a developer might take to customize UI components, as well as the various tradeoffs associated with each method.
 Along the way, we’ll explore how these tradeoffs ultimately led to the solution that we’ve settled on for customizing MUI components.
 
 ## Style modification
 
-If you already use a styling library, don’t hesitate to skip this part.
-See you back in the [Logic modification section](#logic-modification).
+(Already sold on using style libraries? Don’t hesitate to skip this section and move on to [Logic modification](#logic-modification).)
 
 ### Good old CSS
 
 Let's start with the easiest part: modifying the style.
-This will necessarily involve CSS.
-Especially the notion of [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), which says that if an element is targeted by two CSS selectors, the browser will apply the more specific one.
-Most of the time it is the selector using more classes.
+This will necessarily involve CSS—especially the notion of [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), which says that if an element is targeted by two CSS selectors, the browser will apply the more specific one.
+Usually this means that the selector with more classes applied to it is more specific and therefore takes precedence.
 
-For example, if we look at a switch component, we have multiple subcomponents that we could expect to modify.
+For example, if we look at the Material UI `Switch` component, we have multiple subcomponents that we could expect to modify.
 For each of them, we assign a specific CSS class:
 
 <img src="/static/blog/making-customizable-components/switchHighlighted.png" style="width: 532px; margin-top: 16px; margin-bottom: 16px;" alt="Switch component with highlighted sub components" />
 
-Notice that each element is styled using only one CSS class— the thumb style, for example, is applied with the `css-jsexje-MuiSwitch-thumb` class.
-So any CSS selector that includes more than one class will override its style.
+Notice that each element is styled using only one CSS class—the thumb style, for example, is applied with the `css-jsexje-MuiSwitch-thumb` class, so any CSS selector that includes more than one class will override its style.
 
-I’m not a designer, so I made an ugly switch example using only CSS.
-You can play with those examples in a [codesandbox](https://codesandbox.io/s/fast-http-kv85p5?file=/src/App.js)
+I’m not a designer, so I made an ugly switch example using only CSS—
+you can play around with it in [CodeSandbox](https://codesandbox.io/s/fast-http-kv85p5?file=/src/App.js):
```

Comment:
What's the name of this element bewteen "CSS" and "you"?

Reply:
That is every editor's favorite punctuation mark—[the em dash](https://www.thepunctuationguide.com/em-dash.html)! 😁